applications/CoSimulationApplication/custom_external_libraries/CoSimIO/co_sim_io/sources/code_location.cpp

```diff
@@ -24,5 +24,5 @@ std::string CodeLocation::GetCleanFileName() const
     return fs::canonical(fs::path(GetFileName())).lexically_relative(fs::absolute(".")).string();
```

Comment:
@philbucher This isn't changed! Are you sure that you fixed the problem?

Reply:
Yes, the problem was in the implementation of `fs::canonical` in the C++11-compatible version of filesystem that we use

I found a couple of (in the meantime fixed) problems, thus after I updated the lib the problem should be solved.

Ideally you could test it :)

See what I wrote [here](https://github.com/KratosMultiphysics/CoSimIO/pull/352#issuecomment-1709689779)